airsonic-main/src/main/java/org/airsonic/player/validator/UserSettingsValidator.java

```diff
@@ -99,6 +99,9 @@ public void validate(Object obj, Errors errors) {
             }
         }
 
+        if (command.isDeleteUser() && settingsService.getAdminUser().equals(username)) {
+            errors.rejectValue("deleteUser", null, "Cannot delete the primary admin user");
```

Comment:
Maybe we should show the username in the log?

Reply:
Done